app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const {getTopics} = require('./controllers/topics.controllers');
+
+const app = express();
+
+app.get('/api/topics', getTopics);
+
+app.use((req, res, next) => {
+    console.log('my error')
+    res.status(400).send({msg: 'Bad api endpoint'});
+})
```

Comment:
This will be a 404 and you would normally use an app.all here (rather than app.use).